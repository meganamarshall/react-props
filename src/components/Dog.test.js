import React from 'react';
import { shallow } from 'enzyme';
import Dog from './Dog';

describe('Dog component', () => {
  it('matches the snapshot', () => {

    const wrapper = shallow(<Dog name="hello" weight="23 lbs" age={4} />);
    expect(wrapper).toMatchSnapshot();
  });
});
