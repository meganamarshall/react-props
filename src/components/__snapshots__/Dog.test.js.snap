// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dog component matches the snapshot 1`] = `
<dl>
  <dt>
    Name:
  </dt>
  <dd>
    hello
  </dd>
  <dt>
    Age:
  </dt>
  <dd>
    4
  </dd>
  <dt>
    Weight:
  </dt>
  <dd>
    23 lbs
  </dd>
</dl>
`;
